doc/src/hugo/content/guides/authors/design/patterns/command-event/surfaced-parameters.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "4 - Surfaced Parameters"
+description: "Surfacing Entity Data in Events"
+date: 2022-02-25T10:50:32-07:00
+draft: "false"
+weight: 10
+---
+
+# Optional Versioning
+
+## Q: Should the events for each command have only the fields that might have changed, listed individually, when replying to gateway?
+
+### Example:
+
+event UserMembersModified {
+    id: OrganizationId,
+    members: MemberId*
+}
```

Comment:
This one seems kind of short on examples compared to the other cases. Throw a few more events in there.